velox/exec/tests/ContainerRowSerdeTest.cpp

```diff
@@ -144,5 +144,36 @@ TEST_F(ContainerRowSerdeTest, nested) {
   testRoundTrip(nestedArray);
 }
 
+TEST_F(ContainerRowSerdeTest, compareDoubleEquals) {
```

Comment:
In general it seems like all the comparison function tests are done through RowContainerTests. That is the main caller and touch-point for Velox operators. Please can you add tests there also.

Reply:
I will take a look at those tests.